samples: bluetooth: Remove redundant bluetooth LE event handling in main source (#789)
nrfconnect/sdk-nrf-bm: doc/nrf-bm/release_notes/release_notes_changelog.rst

Bluetooth LE samples
--------------------

> * Updated the following samples and applications, that does not support pairing, to call the :c:func:`sd_ble_gatts_sys_attr_set` function only in response to the :c:macro:`BLE_GATTS_EVT_SYS_ATTR_MISSING` event and not as a response to a :c:macro:`BLE_GAP_EVT_CONNECTED` event:

Reviewer: ```suggestion
* Updated the following samples and applications that do not support pairing to call the :c:func:`sd_ble_gatts_sys_attr_set` function only in response to the :c:macro:`BLE_GATTS_EVT_SYS_ATTR_MISSING` event and not as a response to a :c:macro:`BLE_GAP_EVT_CONNECTED` event:
```

Author: Thanks!

   * :ref:`ug_dfu_firmware_loader` (Bluetooth LE)
   * :ref:`ble_lbs_sample`
   * :ref:`ble_nus_sample`
   * :ref:`ble_mcuboot_recovery_entry_sample`

> * Removed the authentication status logging from the following samples and application that does not support pairing (does not use the :ref:`lib_peer_manager` library):

Reviewer: ```suggestion
* Removed the authentication status logging from the following samples and applications that do not support pairing (do not use the :ref:`lib_peer_manager` library):
```

Author: Much appreciated! It seems like I could benefit from going back to school. 😝

   * :ref:`ug_dfu_firmware_loader` (Bluetooth LE)
   * :ref:`ble_lbs_sample`
   * :ref:`ble_nus_sample`
   * :ref:`ble_pwr_profiling_sample`
   * :ref:`ble_mcuboot_recovery_entry_sample`

* :ref:`ble_cgms_sample` sample:

   * Removed:

>       * The call to the :c:func:`sd_ble_gatts_sys_attr_set` function from the main source file.

Reviewer: Can be merged with the bullet point above since there is only one "Remove" item

```suggestion
      * Removed the call to the :c:func:`sd_ble_gatts_sys_attr_set` function from the main source file.
```

Author: Fixed

        The :ref:`lib_peer_manager` library takes care of calling this function when :ref:`lib_peer_manager` is used.

* :ref:`ble_hrs_sample` sample:

   * Removed:

>       * Redundant Logging of authentication status from the main source file.

Reviewer 2: ```suggestion
      * Redundant logging of authentication status from the main source file.
```

Author: Fixed

>       * Redundant Logging of authentication status from the main source file.

Reviewer: same comment as above
```suggestion
      * Removed redundant logging of authentication status from the main source file.
```

Author: Fixed
